src/deadline/client/cli/_groups/attachment_group.py

```diff
@@ -0,0 +1,125 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""
+All the `deadline attachment` commands:
+    * upload
+    * download
+"""
+from __future__ import annotations
+
+import click
+import boto3
+
+from typing import Optional
+
+from .._common import _apply_cli_options_to_config, _handle_error
+from ...config import config_file
+from deadline.client.api._attachment import attachment_download as download_attachment_api
+from deadline.client import api
+from deadline.job_attachments._aws.deadline import get_queue
+from deadline.job_attachments.exceptions import MissingJobAttachmentSettingsError
+from deadline.job_attachments.models import JobAttachmentS3Settings
+
+
+@click.group(name="attachment")
+@_handle_error
+def cli_attachment():
+    """
+    Commands to work with AWS Deadline Cloud Job Attachments.
+    """
+
+
+@cli_attachment.command(
+    name="download",
+    help="BETA",
+)
+@click.option(
+    "--manifests",
+    multiple=True,
+    required=True,
+    help="Comma seaparated File paths to manifest formated files.",
+)
+@click.option(
+    "--s3-root-path", help="Job Attachments S3 root path including bucket name and root prefix."
+)
+@click.option("--path-mapping-rules", help="File to path mapping rules.")
```

Comment:
Is this a path for a file that has path mapping rules? If so, how about a help text like:

```
Path to a file with the path mapping rules to use
```